### ISSUE TITLE
Memcached Caching Core Support

### ISSUE THREAD

**Issue by 7thcubic** [SharpRepository/SharpRepository#198]
I think it is possible for Memcached caching to support .net core as the c# driver has a fork for it

https://github.com/cnblogs/EnyimMemcachedCore

**Comment by omarpiani** [comment]
You are right, the missing of a .NET 4.x version makes me difficult to build a single package. 
I hoped that EnyimMemcached will unify this Core fork .. but this is not done yet.. I'll put hands over it soon

**Comment by 7thcubic** [comment]
@omarpiani 
Also i think something that might be related to caching.

Think it might be a valid caching client. Not entirely sure if it is just a quick replace.

Microsoft.Extensions.Caching.Redis
Microsoft.Extensions.Caching.Memory

ServiceStack.Redis.Core

Thought it might help. Gonna test them out

**Comment by omarpiani** [comment]
I make a multi target version of Memcached  in https://github.com/SharpRepository/SharpRepository/tree/develop-MemcachedCore. 

The big problem is that Core version uses dependency injection of ILoggerFactory and configuration is on appconfig.json.

A good think is use MemcachedCore both Core MVC and MVC5 sample apps. If you can take a look and make a PR!
